pages/common/xzmore.md

```diff
@@ -0,0 +1,10 @@
+# xzmore
+
+> Display text from `xz` or `lzma` compressed files.
+> Almost equivalent to `xzless`, except it respects the `PAGER` environment variable, uses `more` by default and you cannot pass options to the pager.
+> See also: `more`.
```

Comment:
```suggestion
> Almost equivalent to `xzless`, except it respects the `PAGER` environment variable, uses `more` by default and you cannot pass options to the pager. See also: `more`.
```
Thanks for your contribution, In almost all our pages we allow a maximum of 2 command description lines and 1 for more information links (with few notable exceptions for some commands). In this case, I suggest concatenating both lines here.

Reply:
Since the second line is already pretty long, and `more` is already shown there as a command maybe I could remove the third line entirely?

Reply:
```suggestion
> Almost equivalent to `xzless`, except it respects the `PAGER` environment variable, uses `more` by default and you cannot pass options to the pager.
```